### ISSUE TITLE
Use Quick Select Tool programmatically

### ISSUE THREAD

**Issue by BharatAgarSoftTech** [TinyCrayon/TinyCrayon-iOS-SDK#14]
Hello team,
I want to perform  Quick Select programmatically before presenting TCMaskView is it possible?

Thanks.

**Comment by eul** [comment]
Yes, I need this option too. It would be grate to specify some start area for Quick Select programmatically.
